aten/src/ATen/native/quantized/cpu/qembeddingbag_prepack.cpp

```diff
@@ -133,9 +138,32 @@ Tensor qembeddingbag_4bit_prepack(const Tensor& weight) {
   return output;
 }
 
+// Applies 4-bit row-wise quantization by determining the range
+// (maximum - minimum) and offset (minimum value) of each row in the input
+// matrix, and then scaling each element to an 2-bit number between 0 and
+// 15.
+// To later de-quantize values, the scale (range / 15) and zero_point
+// are stored alongside the data. More precisely, each row first has quantized
+// values, and then 2-byte fp16 scale and 2-byte zero_offset.)
+Tensor qembeddingbag_4bit_prepack(const Tensor& weight) {
+  return _qembeddingbag_nbit_prepack_helper(weight, 4 /*BIT_RATE*/);
+}
+
+// Applies 2-bit row-wise quantization by determining the range
+// (maximum - minimum) and offset (minimum value) of each row in the input
+// matrix, and then scaling each element to an 2-bit number between 0 and
+// 3.
+// To later de-quantize values, the scale (range / 3) and zero_point
+// are stored alongside the data. More precisely, each row first has quantized
+// values, and then 2-byte fp16 scale and 2-byte zero_offset.)
+Tensor qembeddingbag_2bit_prepack(const Tensor& weight) {
```

Comment:
Might be worthwhile adding a comment (or TODO) here about the lookup operator not being added yet.